pkg/devserver/service.go

```diff
@@ -119,7 +123,7 @@ func (d *devserver) Run(ctx context.Context) error {
 			addr := fmt.Sprintf("%s:%d", d.Opts.Config.EventAPI.Addr, d.Opts.Config.EventAPI.Port)
 			fmt.Println("")
 			fmt.Println("")
-			fmt.Print(cli.BoldStyle.Render("\tInngest dev server online "))
+			fmt.Print(cli.BoldStyle.Render(fmt.Sprintf("\tInngest %s online ", d.Name())))
```

Comment:
I do with is was `Inngest Lite` and `Inngest Dev Server`, but the `Name()` functions doesn't have a space. Maybe a helper?
 
We can at least make it title case:
```suggestion
			fmt.Print(cli.BoldStyle.Render(fmt.Sprintf("\tInngest %s online ", strings.Title(d.Name()))))
```